src/applications/vaos/project-cheetah/index.jsx

```diff
@@ -39,20 +40,26 @@ export function NewBookingSection({
   const history = useHistory();
   const location = useLocation();
 
+  useEffect(() => {
+    openNewBookingPage(history);
+  }, []);
```

Comment:
This only needs to run the check once, now that the action to fetch the data is separate from the check for when to redirect